corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -54,107 +56,56 @@
 
     var delete_error_message = '{% trans "An error occurred while deleting your location. If the problem persists, please report an issue" %}';
 
-    archive_loc = function(button, name, loc_id) {
-        var archive_location_modal = $('#archive-location-modal')[0];
-
-        function archive_fn() {
-            $(button).disableButton();
-            $.ajax({
-                type: 'POST',
-                url: loc_archive_url(loc_id),
-                dataType: 'json',
-                error: 'error',
-                success: function (response) {
-                    alert_user(archive_success_message({"name": name}), "success");
-                    remove_elements_after_action(button);
-                }
-            });
-            $(archive_location_modal).modal('hide');
-            hqImport('analytix/js/google').track.event('Organization Structure', 'Archive')
-        }
-
-        var modal_context = {
-            "name": name,
-            "loc_id": loc_id,
-            "archive_fn": archive_fn
-        };
-        ko.cleanNode(archive_location_modal);
-        $(archive_location_modal).koApplyBindings(modal_context);
-        $(archive_location_modal).modal('show');
-    };
 
-    unarchive_loc = function(button, loc_id) {
-        $(button).disableButton();
-        $.ajax({
-            type: 'POST',
-            url: loc_unarchive_url(loc_id),
-            dataType: 'json',
-            error: 'error',
-            success: function (response) {
-                remove_elements_after_action(button);
-            }
-        });
+    remove_elements_after_action = function(button) {
```

Comment:
It looks like this function is only ever called from functions you moved into `LocationSearchViewModel`. Probably worth moving it in there as a local function/variable (using `var`) rather than leaving it out here now.